guides/doc-Upgrading_and_Updating/topics/upgrading_content_hosts.adoc

```diff
@@ -28,14 +28,7 @@ If these files have been deleted, they must be restored from a backup in order f
 
 ifdef::satellite[]
 .Procedure
-. In the {ProjectWebUI}, navigate to *Hosts* > *Content Hosts* and select the Content Hosts that you want to upgrade.
-. From the *Select Action* list, select *Manage Repository Sets*.
-. From the *Repository Sets Management* list, select the *{ProjectName} Tools {ProjectVersionPrevious}* checkbox.
-. From the *Select Action* list, select *Override to Disabled*, and click *Done*.
-. When the process completes, on the same set of hosts from the previous steps, from the *Select Action* list, select *Manage Repository Sets*.
-. From the *Repository Sets Management* list, select the *Red Hat {project-client-name}* checkbox.
-. From the *Select Action* list, select *Override to Enabled*, and click *Done*.
-. When the process completes, on the same set of hosts from the previous steps, from the *Select Action* list, select *Manage Packages*.
+. From the *Select Action* list, select *Manage Packages*.
```

Comment:
Isn't the base location missing? What I mean is the `navigate to ... > ...` got removed too.

Reply:
@adamlazik1 Per Evgeni's direction, I have removed the entire section.